hack/e2e.go

```diff
@@ -49,19 +58,11 @@ var (
 		"By default, verify that client and server have exact version match. "+
 		"You can explicitly set to false if you're, e.g., testing client changes "+
 		"for which the server version doesn't make a difference.")
+	checkClusterSize = flag.Bool("check_cluster_size", true, fmt.Sprintf("By default, verify that at there are at least %d nodes running.", minMinionCount))
```

Comment:
`--check_cluster_size=false` was previously in the upstream_k8sm_e2e branch which was merged into this branch.